src/argilla/server/daos/models/records.py

```diff
@@ -56,7 +56,7 @@ class BaseEmbeddingVectorDB(BaseModel):
 
 
 class BaseRecordInDB(GenericModel, Generic[AnnotationDB]):
```

Comment:
Pydantic already casts integers to strings for free. What you mean is to include a custom validator to warn this, right? It can be helpful, but may also fill the server logs with this kind of warnings